server/server.go

```diff
@@ -957,7 +957,7 @@ func (s *Server) GetClusterVersion() semver.Version {
 
 // GetSecurityConfig get the security config.
 func (s *Server) GetSecurityConfig() *grpcutil.SecurityConfig {
-	return &s.cfg.Security
+	return &s.cfg.Security.SecurityConfig
```

Comment:
I think the two `SecutityConfig` need to be distinguished.

Reply:
updated.